classes/class-export-json.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace WP_Stream;
+
+class Export_JSON {
+
+  public function output_file ( $data ) {
```

Comment:
Exporters should also have a `$name` or `$title` property (which is output on the admin screen). There should be an abstract `Exporter` class which each exporter extends (similar to our `Connector` class). The `Export` class should check if each registered exporter is a type of `Exporter`.